platforms/__common.sh

```diff
@@ -51,6 +51,40 @@ shouldOverwriteSdk() {
 	fi
 }
 
+shouldOverwritePlatformSdk() {
+	local platform="$1"
+	result=""
+	case $platform in
+        "php")
+                return [[ "$OVERWRITE_EXISTING_SDKS_PHP" == "true" ]]; echo "$?"
+                ;;
+        "php-composer")
+                return [[ "$OVERWRITE_EXISTING_SDKS_PHP_COMPOSER" == "true" ]]; echo "$?"
+                ;;
+		"java")
```

Comment:
nit: looks like `java`, `nodejs`, `python` and `dotnet` don't line up with the other platforms in this file (maybe an extra tab?)